lib/cli.js

```diff
@@ -12,6 +14,13 @@ const path = require("path");
 const resolveFrom = require("resolve-from");
 const standalone = require("./standalone");
 
+const formatterOptions = _.chain(formatters)
+  .keys()
+  .map(opt => `"${opt}"`)
+  .join(", ")
+  .value()
+  .replace(/, ([a-z"]+)$/u, " or $1");
+
```

Comment:
Rather than defining `formatterOptions` multiple times (above and in `lib/standalone.js`), perhaps making this a util in `lib/utils` with some tests.

The above is not a blocker, but would be nice to have some tests for this.

Reply:
Yes, I think it's a nice idea! I will try to change code and add tests. 💪